Source/Atomic/Graphics/View.cpp

```diff
@@ -1589,8 +1589,12 @@ void View::ExecuteRenderPathCommands()
 
                     SetRenderTargets(command);
 
+                    graphics_->ResetNumPasses();
+
                     for (Vector<LightBatchQueue>::Iterator i = actualView->lightQueues_.Begin(); i != actualView->lightQueues_.End(); ++i)
                     {
+                        graphics_->IncNumPasses();
+
                         // If reusing shadowmaps, render each of them before the lit batches
                         if (renderer_->GetReuseShadowMaps() && NeedRenderShadowMap(*i))
                         {

```

Comment:
// ATOMIC BEGIN, // ATOMIC END block here, please note that you don't need to enclose each line separately, find the best spots for BEGIN and END that encapsulate the changes, in this case left aligning the comments would help as across indent blocks